eng/dependabot/net8.0/Packages.props

```diff
@@ -3,6 +3,15 @@
     Packages in this file have versions updated periodically by Dependabot specifically for .NET 8.
   -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETCore.App.Runtime.win-x64" Version="$(MicrosoftNETCoreApp80Version)" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="$(MicrosoftExtensionsConfigurationAbstractions80Version)" />
```

Comment:
NOTE: I added these in this PR. It looks like this `release/8.0` branch had dependabot define these versions but not actually keep them up-to-date. Let me know if I should revert this section.